token-swap/program/src/processor.rs

```diff
@@ -203,6 +205,17 @@ impl Processor {
             return Err(SwapError::IncorrectPoolMint.into());
         }
 
+        if let Some(fee_constraints) = fee_constraints {
```

Comment:
Hmm, what happens if I just pass `None` into Initialize? Can't I bypass the constraints?
I was expecting to see a `production` feature-wrapped check here, instead of optional passing of the fee_constraints. What am I missing about the way this works?

Reply:
You could potentially find a way to pass `None` into initialize, but only if you can directly call `process_initialize` or `process_with_fee_constraints`.  `process` is the main entrypoint into the program, and it passes the hard-coded `FEE_CONSTRAINTS` into `process_with_fee_constraints`.  I might be missing something about how programs can be used -- is it possible to get around calling `process` in a production context?

Reply:
OH. Apologies, I misread the `process` edit. Disregard!